providers/shared_network.rb

```diff
@@ -3,7 +3,7 @@
 # Cookbook Name:: dhcp
 # Provider:: dhcp_shared_subnet
 #
-# Copyright:: 2015, Chef Brigade
+# Copyright:: 2015-2016, Sous Chefs
```

Comment:
Do we really want copyright headers in every file? If so, we should add them everywhere, elsewise remove them everywhere.